gui/app/main.cpp

```diff
@@ -16,12 +18,37 @@
 #include <iostream>
 #include <memory>
 
+#include "batteryinfo.h"
+#include "simple_clock.h"
+
+QObject *clockInstance(QQmlEngine *engine, QJSEngine *scriptEngine) {
+  Q_UNUSED(engine);
+  Q_UNUSED(scriptEngine);
+  return new SimpleClock();
+}
+
+QObject *batteryInstance(QQmlEngine *engine, QJSEngine *scriptEngine) {
+  Q_UNUSED(engine);
+  Q_UNUSED(scriptEngine);
+  return new BatteryInfo();
+}
+
+void install_fonts() {
+  if (QFontDatabase::addApplicationFont(":/fonts/NotoSans-Regular.ttf") == -1)
+    qWarning() << "Failed to load NatoSans-Regular.ttf";
+
+  if (QFontDatabase::addApplicationFont(":/fonts/Oxygen-Regular.ttf") == -1)
+    qWarning() << "Failed to load Oxygen-Regular.ttf";
+}
+
 int main(int argc, char *argv[]) {
 
   QApplication app(argc, argv);
 
   app.setWindowIcon(QIcon(":/images/Logo.png"));
 
+  install_fonts();
```

Comment:
Should this perhaps happen lower?

Reply:
I don't get what lower means in this context

Reply:
I mean at line 121 or so